### PR TITLE
Fixed host down cases to remove admin client in relevant places and a…

### PR DESCRIPTION
**Pull request by soumyalj**
…dded timeouts

Made some changes in common fixtures by adding some timeouts

@sangeethah : Please review when you get a chance.

### CODE REVIEW

File: tests/v2_validation/cattlevalidationtest/core/test_hostdown.py [hunk 1]
```diff
@@ -379,33 +448,38 @@ def get_containers_on_host_for_service(client, host, service):
...
+    #    assert con.healthState == "unhealthy"
 
     for con in unhealthy_con_list:
         wait_for_condition(
             client, con,
             lambda x: x.state in ('removed', 'purged'),
-            lambda x: 'State is: ' + x.healthState)
-        wait_for_scale_to_adjust(admin_client, service)
+            lambda x: 'State is: ' + x.healthState, timeout=180)
+        wait_for_scale_to_adjust(admin_client, service, timeout=400)
```
Comment (sangeethah) [inline tests/v2_validation/cattlevalidationtest/core/test_hostdown.py]: Why do we need the timeout to be this large ? Can this be parametrized ?

File: tests/v2_validation/cattlevalidationtest/core/test_hostdown.py [hunk 2]
```diff
@@ -353,7 +422,7 @@ def host_down_with_services(admin_client, client, host_down_count,
     wait_for_condition(
         client, service,
         lambda x: x.state == 'active',
-        lambda x: 'State is: ' + x.state)
+        lambda x: 'State is: ' + x.state, timeout=720)
```
Comment (sangeethah) [inline tests/v2_validation/cattlevalidationtest/core/test_hostdown.py]: Can the timeout be parametrized ?

**Comment by soumyalj** [comment]
@sangeethah : Addressed review comments and removed admin_client references. Please review

**Comment by sangeethah** [comment]
LGTM

**Pull request merged**